IncidentFields/incident-phishingsubtype.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id": "incident_subtype",
+    "version": -1,
+    "modified": "2019-08-21T13:40:48.682176491Z",
+    "commitMessage": "",
+    "shouldPublish": false,
+    "shouldCommit": false,
+    "shouldPush": false,
+    "name": "Sub-type",

```

Comment:
Not sure Sub-type is a good name, because it's just for phishing. Maybe "Email Category"? @michalgold what do you think?

Reply:
This is what we decided in the meeting at the design stage